installer/scripts/omsadmin.sh

```diff
@@ -410,7 +364,11 @@ onboard()
     echo "   <FullyQualfiedDomainName>${FQDN}</FullyQualfiedDomainName>" >> $BODY_ONBOARD
     echo "   <EntityTypeId>$AGENT_GUID</EntityTypeId>" >> $BODY_ONBOARD
     echo "   <AuthenticationCertificate>${CERT_SERVER}</AuthenticationCertificate>" >> $BODY_ONBOARD
-    append_telemetry $BODY_ONBOARD
+
+    # append telemetry to $BODY_ONBOARD
+    `$RUBY $TELEMETRY_SCRIPT "$BODY_ONBOARD" "$OS_INFO" "$PROCESS_STATS"`
+    [ $? -ne 0 ] && return 1
```

Comment:
I don't think we want a telemetry failure to break the whole on-boarding.
Maybe just log an error.